src/org/traccar/protocol/EelinkProtocolDecoder.java

```diff
@@ -25,40 +25,55 @@
 import org.traccar.model.CellTower;
 import org.traccar.model.Network;
 import org.traccar.model.Position;
+import org.traccar.helper.Log;
 
 import java.net.SocketAddress;
 import java.util.Date;
 
+import java.nio.charset.StandardCharsets;
+//import org.apache.commons.lang.StringUtils;
+//import org.apache.commons.codec.binary.Hex;
+
 public class EelinkProtocolDecoder extends BaseProtocolDecoder {
 
     public EelinkProtocolDecoder(EelinkProtocol protocol) {
         super(protocol);
     }
 
+    private static final short HEADER_KEY = 0x6767;
+    private int index = 0x0000;
+
     public static final int MSG_LOGIN = 0x01;
     public static final int MSG_GPS = 0x02;
     public static final int MSG_HEARTBEAT = 0x03;
     public static final int MSG_ALARM = 0x04;
     public static final int MSG_STATE = 0x05;
     public static final int MSG_SMS = 0x06;
     public static final int MSG_OBD = 0x07;
+    public static final int MSG_OBDFAULT = 0x09;
     public static final int MSG_DOWNLINK = 0x80;
     public static final int MSG_DATA = 0x81;
 
     public static final int MSG_NORMAL = 0x12;
     public static final int MSG_WARNING = 0x14;
     public static final int MSG_REPORT = 0x15;
     public static final int MSG_COMMAND = 0x16;
+
     public static final int MSG_OBD_DATA = 0x17;
     public static final int MSG_OBD_BODY = 0x18;
     public static final int MSG_OBD_CODE = 0x19;
-    public static final int MSG_CAMERA_INFO = 0x1E;
-    public static final int MSG_CAMERA_DATA = 0x1F;
 
-    private void sendResponse(Channel channel, int type, int index) {
```

Comment:
It is not so much a response as an ACK, sending a response is slightly different and there is more data that a few functions require.
I thought about refactoring this so that either an ACK or response could be in the same function.
I can revert if you want.